### DISCUSSION TITLE
Format persistence file?

### DISCUSSION THREAD

**Discussion by 4iMh9CawHdyu** [ventoy/Ventoy#2153] [General]
Anyone know how to format a persistence .dat file. I;m assuming it won't be as simple as using mkfs since it would turn it into.... e.g. ext4, and also inside the dat file there are two directories (upper and work) so, does anyone know how to format it blank but still useable by ventoy so as not to have to make a new one?

**Comment by 4iMh9CawHdyu** [comment]
After a small search, couldn't anything related to formatting dat files... just erasing them seems to be enough or deleting them completely and creating a new one. My mind quickly went looking for a stronger method (formatting) which apparently wasn't adequate here. So, unless I'm wrong and somebody wants to let me know..... where can I close this conversation?

**Reply by uuaws2** [reply]
your not wrong only deleting the files are the correct way to do it. I think.
